packages/ui/src/1_atoms/Slider/Slider.tsx

```diff
@@ -1,22 +1,54 @@
 import React, { FC } from 'react';
 
+import classNames from 'classnames';
 import ReactSlider from 'react-slider';
 
+import { applyDataAttr } from '../../utils';
 import styles from './Slider.module.css';
 
 type SliderProps = {
   max?: number;
   min?: number;
   step?: number;
-  onAfterChange?: (value: number, thumbIndex: number) => void;
-  onChange?: (value: number, thumbIndex: number) => void;
-  value?: number;
+  value?: number | number[];
+  isDouble?: boolean;
```

Comment:
I don't like this naming, I'd say something like `isSimple` (with the default value being `true`) would be better. What do you think @soulBit?